core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -361,6 +425,52 @@ public void date_sub() {
     assertEquals(missingValue(), eval(dsl.date_sub(nullRef, missingRef)));
   }
 
+  @Test
+  public void date_time() {
```

Comment:
Same as `convert_tz` - could you move tests to a separate file?

Reply:
Why should they be in a different file?